src/main/resources/resources/logisim/strings/analyze/analyze.properties

```diff
@@ -170,6 +170,8 @@ tableExpand = Show All Rows
 tableNoInputs = (no inputs)
 tableNoOutputs = (no outputs)
 tableRowsShown = %d of %d rows shown
+tableZoomIn = Zoom In
+tableZoomOut = Zoom Out
 #
```

Comment:
Could you also please add these two entries in all the other properties-files as:
`# ==> tableZoomIn = `
`# ==> tableZoomOut = `